tensorflow-mixin/config.libsonnet

```diff
@@ -0,0 +1,15 @@
+{
+  _config+:: {
+    dashboardTags: ['tensorflow-mixin'],
+    dashboardPeriod: 'now-1h',
```

Comment:
We normally apply a 30min period as standard to our dashboards.

Reply:
I can make this change.

Reply:
@gaantunes updated